.github/workflows/site.yml

```diff
@@ -96,7 +107,7 @@ jobs:
           SITE="./checkstyle/target/site"
           LINK="https://${{env.AWS_BUCKET_NAME}}.s3.${{env.AWS_REGION}}.amazonaws.com"
           aws s3 cp $SITE s3://${{env.AWS_BUCKET_NAME}}/$FOLDER/ --recursive
-          echo $LINK/$FOLDER/index.html > message
+          echo $LINK/$FOLDER/xdocPath.html#docPath > message
```

Comment:
theoretically it should work but please check i have doubt here
and want to know if any changes in bot's message on some error

Reply:
you should not modify existing message, you need to add additional link to it in case there is a check changes. If there is no check changes, no additional link should be added.

Reply:
ok i thought we use generate site only when there is document change. so i have to add if condition here and other than this, is everything is correct?

Reply:
I doubt. To be honest, testing action is not a trivial thing.
Previous step looks suspicious.
To test your changes, you can do following:
- Use https://github.com/nektos/act (I havent tried it, but if you will it is good)
- Other way is to change script to remove things that are related to upload to S3 (because it requires aws creds) and test your logic on local fork. To do that, you need to commit your changes to your fork master branch, create PR with test branch and generate site so see that links will be generated correctly.

Reply:
ok i will try nektos for now i have added if condition